### DISCUSSION TITLE
Q: BUG？& 报错：waline/vercel@1.31.8: The engine "node" is incompatible with this module. Expected version"≥18".Got "16.20.2"

### DISCUSSION THREAD

**Discussion by baijubaiju233** [walinejs/waline#2310] [Q&A]
### Describe the bug | 问题描述

**报错：waline/vercel@1.31.8: The engine "node" is incompatible with this module. Expected version"≥18".Got "16.20.2"**

在按照新手教程进行vercel部署时，走到这一步：
**环境变量配置完成之后点击顶部的 Deployments 点击顶部最新的一次部署右侧的 Redeploy 按钮进行重新部署。该步骤是为了让刚才设置的环境变量生效。**

发现报错：node版本错误

在github的源码中，package.json的waline/vercel@属性为latest，查看官网后发现也确实最新版本就是1.31.8
但报错为**node版本为16.x**让人疑惑，明明vercel已经不支持18以下的node了

**手动更新waline/vercel的版本为1.31.0后可以正常执行**

不知道是不是bug？求解答

### Website URL | 问题网站

https://vercel.com/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
进设置里看下配置的 Node 版本号呢？
![image](https://github.com/walinejs/waline/assets/424491/6cdc0782-0dc6-4091-a02d-5ee4291c67a4)


---
Go into settings and check the configured Node version number?
![image](https://github.com/walinejs/waline/assets/424491/6cdc0782-0dc6-4091-a02d-5ee4291c67a4)

**Reply by baijubaiju233** [reply]
写的是18.x，要换成20.x吗？

---
It says 18.x, should I change it to 20.x?

**Reply by HeLongaa** [reply]
没有用，三个nodejs版本都会报错

---
It’s useless, all three nodejs versions will report errors

**Reply by xiaoao19** [reply]
遇到同样的问题了

---
Encountered the same problem

**Reply by lizheming** [reply]
@xiaoao19 尝试从头开始重新部署下，目前应该都没有问题了

---
@xiaoao19 Try redeploying from scratch, there should be no problem now
